Files/Assets/terminal/terminal.json

```diff
@@ -5,20 +5,20 @@
     {
       "name": "CMD",
       "path": "cmd.exe",
-      "arguments": "/k \"cd /d {0} && title Command Prompt\"",
+      "arguments": "",
       "icon": ""
     }
     /*
```

Comment:
By removing the comment, what would happen if Powershell is not installed?

Reply:
It is an example profile which users can derive new profiles from.

Reply:
Correct, but by removing the `*/` it adds it is actually adding it as one of the profiles.

Reply:
![image](https://user-images.githubusercontent.com/20365014/85164467-66bd1c80-b232-11ea-93a7-9d7186361bd1.png)